sdv/utils.py

```diff
@@ -15,44 +15,6 @@ def cast_to_iterable(value):
     return [value]
 
 
-def display_tables(tables, max_rows=10, datetime_fmt='%Y-%m-%d %H:%M:%S', row=True):
```

Comment:
Why is this being deleted? Is just not used anymore?

Reply:
Yeah, it's not used anywhere